src/components/date-picker/flatpickr-adapter/flatpickr-adapter.scss

```diff
@@ -15,7 +15,11 @@
 
     * {
         box-sizing: border-box;
-        font-family: 'Roboto';
+        // To ensure that the date-picker's popover is font-agnostic,
+        // but still displays the same font as the rest of the components.
+        // This is needed, since the popover opens in the Portal,
+        // which inherits its font from the root element (Kompendium).
+        font-family: var(--kompendium-example-font-family, inherit);
```

Comment:
I guess that, since we fall back to `inherit`, and we don't seem to expect that anyone should have to set `--kompendium-example-font-family` anywhere, apart from ourselves setting it in the docs, I guess we can live with this if it's _absolutely necessary_. But I really don't like that we are referring to kompendium stuff in our production code. Anything relating to kompendium belongs in the examples or other docs related places only 😞

Reply:
I'm not sure I entirely understand why we can't just use `inherit` here though? Does that inherit the wrong font when we show the component in the docs? 🤔

Reply:
(And if it does, can't that happen for a consumer as well?)

Reply:
Ok, yeah, I've verified that this is needed in order to not get Roboto in the flatpickr in the docs. But it feels like there must be some other way to fix that. I'll have a look.

Reply:
the reason is that in Kompendium, the `--kompendium-example-font-family` is set in deeper container in some shadow dom that displays the "examples". But **_WE_** have this limel-portal that opens in the `body`, which has nothing to do with that deep container. So if we simply say `inherit`, then we will be inheriting from the `body` or `root`, which will give us Kompendium's font family.

Reply:
I added a fixup that solves this with your font-family mixin instead. I've checked that it works for the datepicker, and I would expect that it works for the other cases too. Please have a look and see if you think that's going to work 🙏